modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -280,9 +280,18 @@ function acquia_cms_site_studio_modules_installed($modules, $is_syncing) {
  */
 function acquia_cms_site_studio_css_alter(array &$css) {
   if (\Drupal::service('module_handler')->moduleExists('collapsiblock')) {
+    $module_path = \Drupal::service('extension.list.module')->getPath('collapsiblock');
     // Remove the default css provided by module: `collapsiblock`.
     // It causes issue with site studio template.
-    $path = \Drupal::service('extension.list.module')->getPath('collapsiblock') . '/theme/dist/css/collapsiblock.min.css';
-    unset($css[$path]);
+    // The module uses different paths in different releases. Hence, removing
+    // all possible paths to support all releases of collapsiblock module.
+    $paths = [
+      $module_path . '/theme/dist/css/collapsiblock.min.css',
+      $module_path . '/theme/dist/collapsiblock.css',
+      $module_path . '/theme/dist/style.css',
+    ];
+    foreach ($paths as $path) {
+      unset($css[$path]);
+    }
```

Comment:
Can we use hook_library_info_alter, instead ?. Also I don't see file name `/theme/dist/style.css`